config/docker/build-base/Dockerfile

```diff
@@ -3,6 +3,39 @@ MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
 
 ARG DEBIAN_FRONTEND=noninteractive
 
+# Prepare environment for building packages
+RUN set -ex \
+    && sed -i -- 's/# deb-src/deb-src/g' /etc/apt/sources.list \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends \
+               build-essential \
+               cdbs \
+               devscripts \
+               equivs \
+               fakeroot \
+               wget \
+    && apt-get clean \
+    && rm -rf /tmp/* /var/tmp/*
+
+ADD kmod_kci.patch /root/debs/kmod_kci.patch
+
+# Prepare kmod sources, patch, install dependencies, build
+RUN cd /root/debs && \
+    wget http://deb.debian.org/debian/pool/main/k/kmod/kmod_28.orig.tar.xz && \
+    wget http://deb.debian.org/debian/pool/main/k/kmod/kmod_28-1.debian.tar.xz && \
```

Comment:
How about `apt-get source` instead?

Reply:
Will adjust to apt-get source now, its definitely way better.

Reply:
Done